changelog/fix-7700-card-testing-prevention-prevents-saved-card-payments-on-blocks-checkout

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
+Comment: Fix card testing prevention enabling breaking saved card payments on blocks checkout
```

Comment:
I think this PR would deserve a proper entry into changelog.txt, rather than a comment to skip adding it to the changelog during the release process like this.

Reply:
I think this as a a delicate situation, talking about fraud and mitigations, so I skipped the changelog. If you think it deserves publicity, I can switch it to a visible one.

Reply:
IMO, there is no issue with it since it's a public repo anyway.

Reply:
Updated the changelog in https://github.com/Automattic/woocommerce-payments/pull/7701/commits/3ce6713e425532b95e9a69e0eb006ae041e69ce8.